api-specificatie/openapi.yaml

```diff
@@ -253,12 +241,7 @@ paths:
       operationId: raadpleegAdresseerbaarobjectMetAdoId
       tags:
         - Adresseerbaar object
-      summary: Raadpleeg adresseerbaar object met een identificatie van een adresseerbaar object
-      description: "Raadpleeg het adresseerbaar object met de identificatie verkregen via de zoek-operatie
-        Geleverd worden de huidige gegevens van een adresseerbaarobject evt. met adres en pand.
-        Huidige gegevens zijn actuele gegevens die geen eind status hebben, zoals 'Naamgeving ingetrokken'.
-        Als de nummeraanduiding is ingetrokken, dan levert een bevraging geen adres op.
-        Parameter adresseerbaarobjectidentificatie bevat de 16 cijferige identificatie van een ligplaats, standplaats of verblijfsobject."
+      description: "Raadpleeg een adresseerbaar object met de identificatie van het adresseerbaar object. Dit is de identificatie van een verblijfsobject, ligplaats of standplaats."
```

Comment:
Dit moet dan worden gewijzigd in:
Een adresseerbaar object is een verblijfsobject, ligplaats of standplaats waaraan adressen kunnen worden toegekend.

Reply:
Aangepast